client/main.lua

```diff
@@ -318,25 +328,28 @@ RegisterNetEvent('qb-tow:client:TowVehicle', function()
                                 --remove zone
                                 CurrentLocation.zoneCombo:destroy()
                             end
-                            QBCore.Functions.Notify(Lang:t("mission.vehicle_towed"), "success")
-                        end, function() -- Cancel
+                            exports.qbx_core:Notify(Lang:t("mission.vehicle_towed"), "success")
+                        else 
                             StopAnimTask(cache.ped, "mini@repair", "fixing_a_ped", 1.0)
-                            QBCore.Functions.Notify(Lang:t("error.failed"), "error")
-                        end)
+                            exports.qbx_core:Notify(Lang:t("error.failed"), "error")
+                        end
                     end
                 end
             end
         else
-            QBCore.Functions.Progressbar("untowing_vehicle", Lang:t("mission.untowing_vehicle"), 5000, false, true, {
-                disableMovement = true,
-                disableCarMovement = true,
-                disableMouse = false,
-                disableCombat = true,
-            }, {
-                animDict = "mini@repair",
-                anim = "fixing_a_ped",
-                flags = 16,
-            }, {}, {}, function() -- Done
+            if lib.progressBar({
```

Comment:
Was there a reason for removing the disable movement, mouse, and combat?

Reply:
Same here, you must be in a vehicle.

![image](https://github.com/Qbox-project/qbx_towjob/assets/26857315/11a162f1-c3f2-42b0-afb9-c1f1646e248b)